ScoreSaber/UI/Elements/Profile/ProfileDetailView.cs

```diff
@@ -331,6 +330,11 @@ private void MapPresenceClicked() {
         }
 
         private async Task DownloadOrOpenMap(string hash, int diff) {
+            if(hash.ToCharArray().Length != 40) {
+                Plugin.Log.Error("Hash is not 40 characters long");
```

Comment:
Intended code flow is not an error (make it info with "Hash is not 40 characters long, attempting to load OST/DLC map")

Reply:
fixed